app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,10 @@
 <resources>
     <string name="app_name">ImdbMovieApp</string>
+    <string name="search">Search</string>
+    <string name="cancel">cancel</string>
+    <string name="movies">Movies</string>
+    <string name="the_baby_boss">The Baby Boss</string>
```

Comment:
Do you need this to be here? Shouldn't it be coming from an API?

Reply:
<string name="the_baby_boss">The Baby Boss</string> this coming from API yes, i used it just for xml preview and i think the others are not coming from API. i will check anyway